Gemfile.lock

```diff
@@ -655,12 +656,11 @@ DEPENDENCIES
   thin
   timecop
   tinymce-rails
-  uglifier
   will_paginate
   zip-zip
 
 RUBY VERSION
    ruby 2.3.1p112
 
 BUNDLED WITH
-   1.17.3
```

Comment:
@cterse I see you updated things with bundler v2.3.18. Is there a specific reason for that? I would refrain from doing anything with bundler version change before rigorously testing it first. Can you try with version `1.16.6` or `1.17.3` and let me know if it works?

Reply:
There were some issues with trying to bundle with 1.17.3, and the prompt asked me to install with the later version. I'll try again.